README.md

```diff
@@ -66,24 +66,36 @@ You will need a package on your machine to be able to setup a virtual environmen
 Create your virtual environment.
 the python version should be 3.8
 
-```
-pipenv install -r requirements.dev
+```bash
+pipenv install
 ```
 
 depending on your local python setup you may need to specify the version of python to use for the virtual environment...
 
-```
-pipenv install -r requirements.dev --python 3.8
+```bash
+pipenv install --python 3.8
 ```
 
 #### Your virtual environment should now be activated and ready to startup Wagtail
 
-You should see the virtual environment name before your user account name e.g. (virtual-env-name) user@computer ...
+Enable the new virtual environment:
 
-If not run
+```bash
+pipenv shell
+```
+
+Install required libraries, on Ubuntu 20.04 linux we need to add the `pg_config` util:
 
+```bash
+sudo apt install libpq-dev 
 ```
-pipenv shell
+
+Install the virtual env manager `poetry` from https://python-poetry.org/
```

Comment:
I think this might make `pipenv` redundant, since you can run `poetry shell` to achieve the same thing without adding extra layers.